site/static/examples/admin/cluster-role-and-binding.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kueue-api-get
```

Comment:
Could you please rename this to better reflect its purpose within the Visibility API?